store/resources.js

```diff
@@ -10,6 +10,13 @@ export const useResourcesStore = defineStore("resources", {
                 description:
                     "ISAIC provides local high-performance computing infrastructure that is affordable, tailorable and scalable. Their data sandboxes allow you to experiment and collaborate easily and securely. Through their partnerships and supports, ISAIC connects you to applied knowledge and resources to help move your AI journey forward. ISAIC is enabling experimentation, scale, and powering the A.I.mbition of Western Canadian businesses pursuing AI/ML adoption.",
             },
+            {
+                name: "Work Nicer",
+                logoSvg: "/WorkNicer_9PHPpKJxJ.png",
```

Comment:
oh I did't realize I left the weird characters after. you should just be able to use "WorkNicer.png" now, I fixed it